tracer/test/test-applications/azure-functions/Samples.AzureFunctions.V4Isolated.AspNetCore.SdkV1/Samples.AzureFunctions.V4Isolated.AspNetCore.SdkV1.csproj

```diff
@@ -0,0 +1,35 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>net6.0;net7.0;net8.0</TargetFrameworks>
```

Comment:
Is this correct, we can't/shouldn't run SdkV1 on .NET 9?

Reply:
Correct, you get a "Invalid combination of TargetFramework and AzureFunctionsVersion is set." emitted from the .targets as it doesn't include .NET 9.0 in it.

Reply:
I added a comment to clarify that on both the projects